src/__test__/mountable.test.js

```diff
@@ -20,23 +21,29 @@ describe(`emitMount`, () => {
     document.body.removeChild(node)
   })
 
-  it(`does not emit a mount event if it is not mounted`, () => {
+  it(`does not emit a mount event if it is not mounted`, done => {
     const node = document.createElement(`div`)
 
     // ensure that callback does not get called
-    const callback = () => assert.notOk(true)
+    const callback = sinon.spy()
```

Comment:
Thanks! The trick is we don't want the callback to be called, but AFAICT, mocha expects `done` to be called with or without an error. I initially implemented as `const callback = done(new Error('...'))` but was seeing errors due to `done` not being called.